application/F3DOptionsTools.cxx

```diff
@@ -0,0 +1,565 @@
+#include "F3DOptionsTools.h"
+
+#include "F3DConfig.h"
+#include "F3DException.h"
+#include "F3DPluginsTools.h"
+#include "F3DSystemTools.h"
+
+#include "engine.h"
+#include "interactor.h"
+#include "log.h"
+#include "utils.h"
+
+#include <cassert>
+#include <filesystem>
+#include <iomanip>
+#include <set>
+#include <sstream>
+
+// This is required to avoid cxxopts
+// splitting input positional option by commas
+#define CXXOPTS_VECTOR_DELIMITER '\0'
```

Comment:
Can we define it in CMake instead? For some reason, I dislike defining variable before including file.